articles/azure-functions/durable/durable-functions-checkpointing-and-replay.md

```diff
@@ -124,15 +124,7 @@ The replay behavior creates constraints on the type of code that can be written
 
   If orchestrator code needs to get the current date/time, it should use the [CurrentUtcDateTime](https://azure.github.io/azure-functions-durable-extension/api/Microsoft.Azure.WebJobs.DurableOrchestrationContext.html#Microsoft_Azure_WebJobs_DurableOrchestrationContext_CurrentUtcDateTime) (.NET) or `currentUtcDateTime` (JavaScript) API, which is safe for replay.
 
-  If orchestrator code needs to generate a random GUID, it should use the [NewGuid](https://azure.github.io/azure-functions-durable-extension/api/Microsoft.Azure.WebJobs.DurableOrchestrationContext.html#Microsoft_Azure_WebJobs_DurableOrchestrationContext_NewGuid) (.NET) API, which is safe for replay, or delegate GUID generation to an activity function (JavaScript), as in this example:
-
-  ```javascript
```

Comment:
@brandonh-msft did this JavaScript example get removed on purpose?

Reply:
I did it as part of [this comment from Chris](https://github.com/MicrosoftDocs/azure-docs/pull/35972#discussion_r309489505) but negated to add the bit about `newGuid()` for JavaScript, it seems.